go/bind/keybase.go

```diff
@@ -285,7 +285,11 @@ func LogSend(statusJSON string, feedback string, sendLogs, sendMaxBytes bool, ui
 	logSendContext.InstallID = env.GetInstallID()
 	logSendContext.StatusJSON = statusJSON
 	logSendContext.Feedback = feedback
-	logSendContext.Logs.Desktop = uiLogPath
+	if len(uiLogPath) == 0 {
```

Comment:
this should be `!=` right? but i think we should just drop this param entirely and let the service run the show

Reply:
oops yes. cool will do!